mRemoteV1/Config/Serializers/XmlConnectionsDeserializer.cs

```diff
@@ -487,6 +487,8 @@ private ConnectionInfo GetConnectionInfoFromXml(XmlNode xmlnode)
                 {
                     connectionInfo.SoundQuality = (ProtocolRDP.RDPSoundQuality)Tools.MiscTools.StringToEnum(typeof(ProtocolRDP.RDPSoundQuality), Convert.ToString(xmlnode.Attributes["SoundQuality"].Value));
                     connectionInfo.Inheritance.SoundQuality = bool.Parse(xmlnode.Attributes["InheritSoundQuality"].Value);
+                    connectionInfo.RDPMinutesToIdleTimeout = Convert.ToInt32(xmlnode.Attributes["RDPMinutesToIdleTimeout"].Value);
+                    connectionInfo.Inheritance.RDPMinutesToIdleTimeout = bool.Parse(xmlnode.Attributes["InheritRDPMinutesToIdleTimeout"].Value);
```

Comment:
Yea, pretty much. You can even inline this by using the null conditional operator and then the null coalescing operator.

```csharp
collectionOfInts["abc"]?.Value ?? 0
```